grpc/src/client/load_balancing/child_manager.rs

```diff
@@ -305,3 +460,32 @@ impl WorkScheduler for ChildWorkScheduler {
         }
     }
 }
+
+struct RoundRobinPicker {
+    pickers: Vec<Arc<dyn Picker>>,
+    next: AtomicUsize,
+}
+
+impl RoundRobinPicker {
+    fn new() -> Self {
+        Self {
+            pickers: vec![],
+            next: AtomicUsize::new(0),
+        }
+    }
+
+    fn add_picker(&mut self, picker: Arc<dyn Picker>) {
+        self.pickers.push(picker);
+    }
+}
+
+impl Picker for RoundRobinPicker {
+    fn pick(&self, request: &Request) -> PickResult {
+        let len = self.pickers.len();
+        if len == 0 {
+            return PickResult::Queue;
+        }
```

Comment:
Can this special case be handled by having a queuing picker instead, and not using this with empty lists?